lib/js/jsffi.nim

```diff
@@ -184,6 +184,7 @@ proc `and`*(x, y: JsObject): JsObject {.importcpp: "(# && #)".}
 proc `or` *(x, y: JsObject): JsObject {.importcpp: "(# || #)".}
 proc `not`*(x:    JsObject): JsObject {.importcpp: "(!#)".}
 proc `in` *(x, y: JsObject): JsObject {.importcpp: "(# in #)".}
+proc `**` *(x, y: SomeNumber): SomeNumber {.importcpp: "((#) ** #)".}
```

Comment:
What? Originally you did this on `JsObject`, now it is on ``SomeNumber``. If you just want to provide the nim mapping of what is a builtin in javascript, you have to use JsObject everywhere, here as well.

If you want to provide exponentiation, we already have ``math.pow``.

Reply:
I don't understand this feedback. Exponentiation does not work on `JsObject`. In javascript you do not exponentiate two objects, you exponentiate two numbers.